addons/main/functions/Profiles/fnc_doesProfileAllow.sqf

```diff
@@ -0,0 +1,41 @@
+#include "script_component.hpp"
+/*
+ * Author: joko // Jonas
+ * Group profiles handler -- Checks and runs AI profiles -- group actions with the fnc_profileXxx prefix.
+ *
+ * Arguments:
+ * 0: Group/Unit <OBJECT, GROUP>
+ * 1: Profile Tactic <STRING>
+ *
+ * Return Value:
+ * Tactic is allowed to Execute
+ *
+ * Example:
+ * [bob] call lambs_danger_fnc_doesProfileAllow;
+ *
+ * Public: No
+*/
+params [["_target", objNull, [objNull, grpNull]], ["_tactic", "unkown", [""]]];
+
+private _profile = _target getVariable [QGVAR(AIProfile), "default"];
```

Comment:
Kind of a QoL thing. Have a cba setting which profile should be used as default

the CBA setting is a list of available profiles

Reply:
a dropdown would be impossible because they can be saved in the description and created via scripted modules. and parsing that while in the editor is from my knowledge not possible. the only thing for CBA setting we could do is a normal text box.

Reply:
The way I see it, is, a drop down is doable, but only account for config and missionconfig profiles. If someone adds them in editor OR via script it is in their authority to set the default correctly.